src/assets/fonts.tsx

```diff
@@ -0,0 +1,11 @@
+import { injectGlobal } from 'styled-components'
+
+injectGlobal`
+    @font-face {
+        font-family: 'artsy-icons';
```

Comment:
I switched the logotype font-face icon with the svg per @craigs suggestion just to see what that might look like. @l2succes @alloy are there any strong opinions about which implementation we should go with? I can switch this back to go ahead and get this merged and we can decide later...